### DISCUSSION TITLE
✅ "Nothing Found" message for loop grid, if there is no posts in a category for example

### DISCUSSION THREAD

**Discussion by As3ad10** [elementor/elementor#25853] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Imagine I made a loop grid for search results but no results found. what is showing now is a nothing, no message and no feedback. That makes the visitor confused and frustrated.  

### Describe the solution you'd like.

It would be nice if there is a "Nothing found" message for the loop. Just like the "Archive Posts" element that have the "nothing found message". 

### Describe alternatives you've considered.

It can also be solved by adding the ability to choose a loop item in the "Archive Posts" element skin. This would be a great feature.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.18.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
